### ISSUE TITLE
"Most popular" packages

### ISSUE THREAD

**Issue by dalyUSN** [umbraco/Umbraco.Marketplace.Issues#32]
At the moment, it seems like the "Most popular" packages row on the homepage is based on historical downloads. The most popular historically can be misleading because I presume it rolls in V7 and V8 downloads too.

Would it make more sense for this to be based on a combination of factors? 

For instance, the number of downloads for the most recent version of Umbraco — maybe even the two most recent versions of Umbraco.

If others do not agree with this then perhaps there could be a new row for "Most Popular Umbraco 11" packages.

It would be good to know others' thoughts on this.

I'm hoping with this suggestion we can make the Marketplace more relevant.

**Comment by nathanwoulfe** [comment]
Hey @dalyUSN, this is absolutely on our radar - we're looking into how we can integrate and make use of the package telemetry data to better indicate popular packages by install rather than by download. As you rightly pointed out, the historic NuGet downloads is not a great measure, but was all we had at the start of the project.

**Comment by nathanwoulfe** [comment]
Hi @dalyUSN, I think we can safely close this one now - we've integrated the available package telemetry data and updated the 'most popular' listings to be ordered based on the number of package installs reported in the previous 14 days. This keeps the data somewhat dynamic (certainly more so than using the NuGet downloads), and gives a much more realistic view of current package popularity.

Of course this is reliant on packages enabling telemetry collection, but hopefully this change is something of an incentive.

There's also scope for future improvements to how we calculate rankings - we could, for example, weight recent/latest Umbraco versions higher.

For now though, the main issue here has been resolved 👍
